weaviate/graphql/neartextbuilder.go

```diff
@@ -166,3 +168,48 @@ func (e *NearTextArgumentBuilder) build() string {
 	}
 	return fmt.Sprintf("nearText:{%v}", strings.Join(clause, " "))
 }
+
+func (e *NearTextArgumentBuilder) togrpc() *pb.NearTextSearch {
+	nearText := &pb.NearTextSearch{
+		Query: e.concepts,
+	}
+	if e.withCertainty {
+		certainty := float64(e.certainty)
+		nearText.Certainty = &certainty
+	}
+	if e.withDistance {
+		distance := float64(e.distance)
+		nearText.Distance = &distance
+	}
+	if e.moveTo != nil {
+		nearText.MoveTo = e.parseMoveParam(e.moveTo)
+	}
+	if e.moveAwayFrom != nil {
+		nearText.MoveAway = e.parseMoveParam(e.moveAwayFrom)
+	}
+	if e.targets != nil {
+		nearText.Targets = e.targets.togrpc()
+	}
+	if len(e.targetVectors) > 0 && e.targets == nil {
+		nearText.Targets = &pb.Targets{TargetVectors: e.targetVectors}
+	}
+	return nearText
+}
+
+func (e *NearTextArgumentBuilder) parseMoveParam(moveParam *MoveParameters) *pb.NearTextSearch_Move {
```

Comment:
```suggestion
func (e *NearTextArgumentBuilder) buildMoveParam(moveParam *MoveParameters) *pb.NearTextSearch_Move {
```

suggestion: perhaps "build" is more appropriate in this context